functions/src/Services/Elections.ts

```diff
@@ -24,4 +26,21 @@ export const openElections = () => {
 		})
 		.then(() => Promise.resolve())
 		.catch(error => Promise.reject(error));
+};
+
+export const addApplication = (candidate: Candidate) => {
+	const positionCollection = getPositionCollection();
+
+	positionCollection.doc(candidate.applyPosition).collection('candidates')
+	.add({ ...candidate })
```

Comment:
this tabbing looks kinda weird. don't understand why it's the only line that is so far to the left. It looks like the standard you guys have is method calls are 1 tab in from it's parent.

Reply:
I changed the tabs, let me know if I had the correct interpretation of what you asked for.

Reply:
yeah this looks good